src/main/java/com/epam/javacourses2016/task10/SolverTask10.java

```diff
@@ -17,6 +22,29 @@
      * @return Множество пар <слово, количество вхождений в файле>.
      */
     public HashMap<String, Integer> countNumberWords(File input) {
-        return null;
+
+        HashMap<String, Integer> englishWords = new HashMap<>();
+        try (FileReader fr = new FileReader(input); BufferedReader br = new BufferedReader(fr)) {
```

Comment:
`fr` is never used, wrap it in one line like `BufferedReader br = new BufferedReader(new FileReader(input))`